usage/usage.go

```diff
@@ -158,18 +159,24 @@ func poll(pollingInterval time.Duration) error {
 		float64(allStat.CPUStatAll.Idle-lastAllStat.CPUStatAll.Idle)
 
 	relativeCPU := selfCPU / allCPU
```

Comment:
I would really prefer you check allCPU = 0 rather than using IsNaN?  Something along the lines of

```suggestion
	if allCPU > 0.0 {
	    relativeCPU = selfCPU / allCPU
	} else {
	    relativeCPU = 0.0
	}
```